src/main/scala/shell/xilinx/ArtyShell.scala

```diff
@@ -170,7 +170,7 @@ abstract class ArtyShell(implicit val p: Parameters) extends RawModule {
         dut.qspi(0),
         dut.clock,
         dut.reset,
-        syncStages = qspi_params.sampleDelay
+        syncStages = 0
```

Comment:
why did you remove the syncstages? I would have thought you would have set this to syncStages = qspi_params.defaultSampleDel

Reply:
Agree. This was an old change, I needed to revert this to the default setting, as was done in other config files.

Reply:
updated.